src/main/java/com/polyscripts/contactManagement/repos/ContactEmployeeRepo.java

```diff
@@ -0,0 +1,7 @@
+package com.polyscripts.contactManagement.repos;
```

Comment:
please use explicite package names, in this case "repository" instead of "repos"
by convention, packages must be singular (controller, service ....), not like node js projects in witch developers makes thier own patterns ;)